forge-gui/res/tokenscripts/w_1_1_human_noble_vanishing_prevention.txt

```diff
@@ -0,0 +1,8 @@
+Name:Human Noble
```

Comment:
wrong names

Reply:
Do all tokens need the _token_ at the end ? 

![image](https://github.com/Card-Forge/forge/assets/67333662/dad9789b-0cc3-47c1-b33a-1be0691984e5)

Reply:
> 111.4. A spell or ability that creates a token sets both its name and its subtype(s). If the spell or ability doesn't specify the name of the token, its name is the same as its subtype(s) plus the word "Token."